test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -14,14 +14,48 @@ namespace KozaniManagerTestPackage
     constexpr PCWSTR c_PackageFullName = L"KozaniManager_1.0.0.0_neutral__8wekyb3d8bbwe";
 }
 
+HRESULT RegisterDVCPluginIfNeeded()
+{
+    PCWSTR subkey = L"Software\\Microsoft\\Terminal Server Client\\Default\\AddIns\\KozaniDVC";
+    PCWSTR keyname = L"Name";
+
+    wil::unique_hkey key;
+    LSTATUS result = ::RegOpenKey(HKEY_CURRENT_USER, subkey, &key);
+    if (result == ERROR_SUCCESS)
+    {
+        WEX::Logging::Log::Comment(L"DVC plugin key already exists!");
+        return S_OK;
+    }
+
+    PCWSTR kozaniDvcClientClsid{ L"ed202fc8-ee21-4bf2-9681-23a8e97be9f3" };
+    WCHAR regCommand[MAX_PATH];
+    swprintf_s(regCommand, L"add \"HKCU\\%s\" /v %s /t REG_SZ /d {%s}", subkey, keyname, kozaniDvcClientClsid);
+    HINSTANCE hInst = ShellExecute(nullptr, nullptr, L"reg.exe", regCommand, nullptr, SW_SHOW);
+    WEX::Logging::Log::Comment(WEX::Common::String().Format(L"ShellExecute reg add finished with code: %d", hInst));
+
+    if ((int)hInst > 32)
+    {
+        WEX::Logging::Log::Comment(L"Successfully created DVC plugin key!");
+    }
+    else
+    {
+        HRESULT hr = HRESULT_FROM_WIN32((DWORD)hInst);
```

Comment:
`const HRESULT hr{ HRESULT_FROM_WIN32(static_cast<DWORD>(hInst));`

New-style cast notation is recommended over old-style C casts.

Also, [ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)